src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -1,15 +1,20 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Truck
- * Do not remove no-field constructor
- */
 public class Truck extends Machine {
+    private int loadCapacity;
```

Comment:
The `loadCapacity` field should have an access modifier to encapsulate the field properly. Consider using 'private' to ensure that this field is not directly accessible from outside the class, and provide getters and setters for access.